plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/controller/chat/telemetry/TelemetryHelper.kt

```diff
@@ -3,15 +3,19 @@
 
 package software.aws.toolkits.jetbrains.services.cwc.controller.chat.telemetry
 
+import org.assertj.core.util.VisibleForTesting
```

Comment:
? wrong annotation

this library shouldnt even be available in this source context